dms-core/src/main/kotlin/team/aliens/dms/domain/vote/usecase/UpdateVotingTopicUseCase.kt

```diff
@@ -0,0 +1,39 @@
+package team.aliens.dms.domain.vote.usecase
+
+import team.aliens.dms.common.annotation.UseCase
+import team.aliens.dms.common.spi.TaskSchedulerPort
+import team.aliens.dms.domain.notice.service.CommandNoticeService
+import team.aliens.dms.domain.vote.dto.request.UpdateVotingTopicRequest
+import team.aliens.dms.domain.vote.exception.NotValidPeriodException
+import team.aliens.dms.domain.vote.service.VoteService
+import java.time.LocalDateTime
+
+@UseCase
+class UpdateVotingTopicUseCase(
+    private val taskSchedulerPort: TaskSchedulerPort,
+    private val voteService: VoteService,
+    private val noticeService: CommandNoticeService
+) {
+
+    fun execute(request: UpdateVotingTopicRequest) {
+        if (request.startTime.isAfter(request.endTime) || request.endTime.isBefore(LocalDateTime.now())) {
+            throw NotValidPeriodException
+        }
+
+        val votingTopic = voteService.getVotingTopicById(request.id)
+        var isReNotice = if (votingTopic.endTime.isBefore(LocalDateTime.now())) true else false
```

Comment:
```suggestion
        val isReNotice = votingTopic.endTime.isBefore(LocalDateTime.now())
```
isBefore() 메서드는 Boolean을 반환합니다...

Reply:
수정했습니다.

Reply:
val 로 선언해주세요

Reply:
수정 완료했습니다.